doc/server_configurations.md

```diff
@@ -7695,14 +7695,50 @@ Volar can be installed via npm
 npm install -g @volar/server
 ```
 
-With Vue 3 projects - it works out of the box.
+Vue 3 works by default, but Vue 2 needs [additional configuration](https://github.com/johnsoncodehk/volar/blob/master/extensions/vscode-vue-language-features/README.md?plain=1#L28-L63).
```

Comment:
can you change "works by default" to something more explicitly. Works with volar I'd guess.

Reply:
Changed to

Volar in Vue 3 projects doesn't need any additional configuration and works out of the box. Vue 2 projects need [additional configuration](https://github.com/johnsoncodehk/volar/blob/master/extensions/vscode-vue-language-features/README.md?plain=1#L28-L63).